.env.example

```
@@ -45,8 +45,18 @@ BUNQUEUE_PORT="6789"
 RESEND_API_KEY=""
 EMAIL_FROM_ADDRESS="noreply@curiositi.com"
 
+# AI Providers - at least one is required for agent chat
+OPENAI_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The OPENAI_API_KEY key is duplicated

(DuplicatedKey)